test/pylib/manager_client.py

```diff
@@ -66,86 +66,92 @@ async def before_test(self, test_name: str) -> None:
         dirty = await self.is_dirty()
         if dirty:
             self.driver_close()  # Close driver connection to old cluster
-        await self._request(f"/cluster/before-test/{test_name}")
+        await self._get(f"/cluster/before-test/{test_name}")
         if self.cql is None:
             # TODO: if cluster is not up yet due to taking long and HTTP timeout, wait for it
             # await self._wait_for_cluster()
             await self.driver_connect()  # Connect driver to new cluster
 
     async def after_test(self, test_name: str) -> None:
         """Tell harness this test finished"""
-        await self._request(f"/cluster/after-test/{test_name}")
+        await self._get(f"/cluster/after-test/{test_name}")
 
-    async def _request(self, resource: str) -> str:
+    async def _get(self, resource: str) -> aiohttp.ClientResponse:
```

Comment:
I've read `aiohttp` had issues if the `resp.text()` was not extracted. Let's keep an eye on that in case of failures.